setup.cfg

```
@@ -0,0 +1,73 @@
+[metadata]
+name = stepwise
+description = A Python-based Trajectory Optimization Framework
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Stefano Dafarra
+author_email = stefano.dafarra@iit.it
+license = BSD-2
+license_files = LICENSE
+platforms = any
+url = https://github.com/ami-iit/stepwise
+
+project_urls =
+    Changelog = https://github.com/ami-iit/stepwise/releases
+    Source = https://github.com/ami-iit/stepwise
+    Tracker = https://github.com/ami-iit/stepwise/issues
+
+keywords =
+    trajectory
+    optimization
+    robots
+    humanoids
+    quadrupeds
+    kinematics
+    dynamics
+    contacts
+
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Framework :: Robot Framework
+    Intended Audience :: Science/Research
+    Intended Audience :: Developers
+    Intended Audience :: Education
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Operating System :: POSIX :: Linux
+    Operating System :: MacOS
+    Operating System :: Microsoft :: Windows
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Games/Entertainment :: Simulation
+
+[options]
+zip_safe = False
+packages = find:
+package_dir =
+    =src
+python_requires = >=3.8
```

I would support only Python > 3.10 since the idea is to heavily rely on `dataclasses` and 3.10 significantly improves inheritance properties that we surely need.

```suggestion
python_requires = >=3.10
```

---

Now that this means not supporting Ubuntu 20.04 with apt dependencies (and I like it).